BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+## @ GenerateKeys.py
+#
+# This script is an reference to generate RSA private keys required by slimbootloader.

```

Comment:
This script is a reference to generate 'test' RSA private keys required by slimbootloader.

Reply:
Sure. Will update.

Reply:
Done